examples/ember-cli/stories/ember-typescirpt.stories.ts

```diff
@@ -0,0 +1,27 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { Story, Meta } from '@storybook/ember/types-6-0';
+import type { ButtonArgs } from '../app/components/button';
+
+export default {
+  title: 'TypeScript',
+} as Meta;
+
+const Template: Story<ButtonArgs> = (args) => ({
+  template: hbs`
+  <Button @emoji={{this.emoji}} @onClick={{this.onClick}}>
```

Comment:
```suggestion
  <Button @emoji={{this.emoji}} {{on "click" this.onClick}}>
```

Reply:
The purpose of this story, and the TypeScript changes, is to show how `ButtonArgs` from the glimmer component can also be used as the `Args` type for the story.

I can make this change as the `emoji` property still shows this point